sqlglot/dialects/bigquery.py

```diff
@@ -903,6 +904,7 @@ class Generator(generator.Generator):
             exp.GenerateSeries: rename_func("GENERATE_ARRAY"),
             exp.GroupConcat: rename_func("STRING_AGG"),
             exp.Hex: lambda self, e: self.func("UPPER", self.func("TO_HEX", self.sql(e, "this"))),
+            exp.HexString: lambda self, e: self.hexstring_sql(e, binary_function_repr=True),
```

Comment:
I suggest preserving 0xff in BigQuery and ClickHouse, this will convert it into 255.